expression-language.md

```diff
@@ -182,10 +182,10 @@ The type system contains 3 _primitive_ types:
   32-bit, twos-complement encoding.
 - _Boolean_: A boolean value of "true" or "false".
 
-The [types defined in the CloudEvents specification][ce-type-system] URI, URI Reference and Timestamp are represented as
+The [types defined in the CloudEvents specification][ce-type-system]-- URI, URI Reference, and Timestamp-- are represented as
```

Comment:
why "--" instead of just a comma? Looks a bit odd to me
![image](https://user-images.githubusercontent.com/1944671/112238535-379d6680-8c1b-11eb-8607-c6873e855622.png)

Reply:
or parens

Reply:
Parens lgtm.